configs/config_sandbox/templates/train-small-3class-template.yaml

```diff
@@ -1,11 +1,13 @@
 train_config:
-  model_id_prefix: 'segmentation-model-small-3class'
-  dataset_id: 'dataset-small-3class-template'
+  model_id_prefix: 'segmentation-model-small-3class-template'
+  dataset_id: 'dataset-small-3class_tfversion_debug'
```

Comment:
I used this dataset because it works but maybe you should create a `dataset-small-3class-template` one as you were doing before?

Reply:
yes, do this if not done